account_financial_report_qweb/report/general_ledger.py

```diff
@@ -890,6 +892,28 @@ def _inject_line_not_centralized_values(
     m.name AS entry,
     j.code AS journal,
     a.code AS account,
+    CASE
+        WHEN
+            ml.tax_line_id is not null
+        THEN
+            at.name
+        WHEN
+            (SELECT count(*)
+            FROM account_move_line_account_tax_rel aml_at_rel
+            WHERE aml_at_rel.account_move_line_id = ml.id
+            LIMIT 1) > 0
```

Comment:
I think this WHEN on select count is unnecessary because `array_to_string(array_agg(at.name), ', ')` will return an empty string when there is no record. So it should be more performant without.

Reply:
Indeed, I updated my code.